build-patched-kao-kcmo-images.sh

```diff
@@ -137,10 +140,10 @@ function update_base_image() {
 
     rhpkg clone containers/${brew_repo}
     pushd ${brew_repo}
-    git checkout --track origin/crc-1-rhel-8
-    base_image_of_repo=$(grep "^FROM openshift/ose-base" Dockerfile | sed 's/^FROM //')
+    git checkout --track origin/crc-1-rhel-9
```

Comment:
Same comment here, we don't *have* to switch to `crc-1-rhel-9` in order to change the base image used in the Dockerfiel.